source/Pages/compute_on_spider.rst

```diff
@@ -305,7 +318,408 @@ combination that is not available you will receive the following error message:
+
+
+The example we are about to execute in the container comes from the tensorflow library: `classifying pieces of clothing <https://www.tensorflow.org/tutorials/keras/classification>`_. Now create a file to run ``fashion.py``, set it to executable with ``chmod 755 fashion.py`` and add the following:
+
+.. code-block:: python
+
+  #!/usr/bin/env python
+
+  # TensorFlow and tf.keras
+  import tensorflow as tf
+
+  # Helper libraries
+  import numpy as np
+  import matplotlib.pyplot as plt
+
+  print(tf.__version__)
+
+  fashion_mnist = tf.keras.datasets.fashion_mnist
+
+  (train_images, train_labels), (test_images, test_labels) = fashion_mnist.load_data()
+
+  class_names = ['T-shirt/top', 'Trouser', 'Pullover', 'Dress', 'Coat',
+                 'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
+
+  train_images = train_images / 255.0
+  test_images = test_images / 255.0
+
+  model = tf.keras.Sequential([
+      tf.keras.layers.Flatten(input_shape=(28, 28)),
+      tf.keras.layers.Dense(128, activation='relu'),
+      tf.keras.layers.Dense(10)
+  ])
+
+  model.compile(optimizer='adam',
+                loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+                metrics=['accuracy'])
+
+  model.fit(train_images, train_labels, epochs=10)
+
+  test_loss, test_acc = model.evaluate(test_images,  test_labels, verbose=2)
+  print('\nTest accuracy:', test_acc)
+
+  probability_model = tf.keras.Sequential([model,
+                                           tf.keras.layers.Softmax()])
+
+  predictions = probability_model.predict(test_images)
+  print(predictions[0])
+
+This example will create a model that recognizes the clothes in a picture, and a prediction of a set of test images is done at the end. The result can be compared to the `official example <https://www.tensorflow.org/tutorials/keras/classification>`_. The matplotlib output is omitted in this example for simplicity. This output can be seen in the section on :ref:`jupyter notebooks <jupyter-notebooks>`.
+
+Now this code can be run with:
+
+.. code-block:: bash
+
+   singularity exec --nv nv-tf-22.07.sif ./fashion.py
+
+Or run it interactively in the container line-by-line with:
+
+.. code-block:: bash
+
+   singularity shell --nv nv-tf-22.07.sif 
+
+If there is an output in the terminal running the python code similar to:
+
+.. code-block:: bash
+
+   2022-07-29 11:53:24.017428: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1532] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 30987 MB memory:  -> device: 0, name: Tesla V100-PCIE-32GB, pci bus id: 0000:00:06.0, compute capability: 7.0
+
+this means the GPU is being used for your computations.
+
+Also, by wrapping the singularity command in a shell script called ``fashion.sh`` and adding the appropriate ``#SBATCH`` commands at the top, the script can be submitted to the batch system with ``sbatch fashion.sh``.
```

Comment:
I cant see it here, but are you planning to link this script or just name it? Linking it as an example would be nice.